powerauth-webflow-i18n/src/main/java/io/getlime/security/powerauth/app/webflow/i18n/I18NService.java

```diff
@@ -68,4 +74,19 @@ public AbstractMessageSource getMessageSource() {
         return messageSource;
     }
 
+    /**
+     * Get language setting json file.
+     *
+     * @return content of the file stripped of new lines.
+     */
+    public String readLanguageSetting() {
+        try {
+            // language setting is possible only via extension and external resources
+            return new String(Files.readAllBytes(languageSettingSource.getFile().toPath()));
+        } catch (IOException ex) {
+            // language setting is not configured return null
```

Comment:
Let's log it please, at least for `DEBUG`.

Reply:
logger added